mods/content/pixel_shifting/pixel_shifting.dm

```diff
@@ -0,0 +1,44 @@
+/mob
+	/// Wether the mob is currently pixel shifted or not
+	var/is_shifted
+	/// If the mob is currently pixel shifting
+	var/shifting
+
+/mob/proc/unpixel_shift()
+	return
+
+/mob/living/unpixel_shift()
+	if(is_shifted)
+		is_shifted = FALSE
+		pixel_x = initial(pixel_x)
+		pixel_y = initial(pixel_y)
+
+/mob/proc/pixel_shift(direction)
+	return
+
+/mob/living/pixel_shift(direction)
+	switch(direction)
+		if(NORTH)
+			if(!canface())
```

Comment:
It should be the first check inside this function. No need to duplicate it for every direction.
Use this:
```
/mob/proc/can_pixel_shift()
	return !(incapacitated() || buckled || LAZYLEN(grabbed_by))

/mob/proc/pixel_shift(direction)
	return

/mob/living/pixel_shift(direction)
	if(!canface() || !can_shift())
		return FALSE
	
	(...)